src/manager.rs

```diff
@@ -88,65 +88,65 @@ fn err_json(value: &Value, expected_value: &'static str) -> Error {
 pub struct KeyHolderWrite {
     key: RedisKeyWritable,
     key_name: String,
+    val: Option<*mut RedisJSON>,
```

Comment:
Why pointer? So it doesn't get dropped?

Reply:
Changed to reference with lifetime after a few hours of fighting the compiler :)